compiler/src/main/kotlin/dev/zacsweers/metro/compiler/fir/checkers/AggregationChecker.kt

```diff
@@ -3,6 +3,8 @@
 package dev.zacsweers.metro.compiler.fir.checkers
 
 import dev.drewhamilton.poko.Poko
+import dev.zacsweers.metro.compiler.asName
+import dev.zacsweers.metro.compiler.fir.*
```

Comment:
nit: please match the code style of the project

Reply:
I know, spotless doesn't check this unfortunately. Just asking that you match it anyway :)